File: data_out/two-dim/BNR_1_order/TEST/value_plot.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Mon Apr 23 12:53:01 2018

@author: leixin
"""

import numpy as np
rho  = np.loadtxt('./RHO.csv',  delimiter=',')
u    = np.loadtxt('./U.csv',    delimiter=',')
v    = np.loadtxt('./V.csv',    delimiter=',')
p    = np.loadtxt('./P.csv',    delimiter=',')
phi_a= np.loadtxt('./PHI_a.csv',delimiter=',')
z_a  = np.loadtxt('./Z_a.csv',  delimiter=',')
phi_b= np.loadtxt('./PHI_b.csv',delimiter=',')
z_b  = np.loadtxt('./Z_b.csv',  delimiter=',')
num_x=len(rho[0])
num_y=len(rho)
if not ('d_x' in dir()):
    d_x=0.01
if not ('d_y' in dir()):
    d_y=0.01
X = np.arange(-0.5*num_x, 0.5*num_x, 1)*d_x
Y = np.arange(-0.5*num_y, 0.5*num_y, 1)*d_y
X, Y = np.meshgrid(X, Y)
import matplotlib.pyplot as plt
from mpl_toolkits.mplot3d import Axes3D
'''
plot density 
'''
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, rho, cmap=plt.cm.winter)
plt.savefig("rho.png")
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, p, cmap=plt.cm.winter)
plt.savefig("p.png")
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, u, cmap=plt.cm.winter)
plt.savefig("u.png")
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, v, cmap=plt.cm.winter)
plt.savefig("v.png")
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, phi_a, cmap=plt.cm.winter)
plt.savefig("phi_a.png")
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, phi_b, cmap=plt.cm.winter)
plt.savefig("phi_b.png")
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, z_a, cmap=plt.cm.winter)
plt.savefig("z_a.png")
fig = plt.figure()  
ax = Axes3D(fig)
ax.plot_surface(X, Y, z_b, cmap=plt.cm.winter)
plt.savefig("z_b.png")
#plt.show()

File: data_out/two-dim/BNR_1_order/TEST/value_plot.m
rho  = load('./RHO.csv');
u    = load('./U.csv');
v    = load('./V.csv');
p    = load('./P.csv');
phi_a= load('./PHI_a.csv');
z_a  = load('./Z_a.csv');
phi_b= load('./PHI_b.csv');
z_b  = load('./Z_b.csv');
temper=load('./TEMPER.csv');
d_x=1;
d_y=1;
num_x=size(rho,2);
num_y=size(rho,1);
X = (1:num_x)*d_x;
Y = (1:num_y)*d_y;
[X, Y] = meshgrid(X, Y);
figure(1);
surf(X, Y, rho);
title('rho');
figure(2);
surf(X, Y, p);
title('p');
figure(3);
surf(X, Y, u)
title('u')
figure(4)
surf(X, Y, v);
title('v');
figure(5);
surf(X, Y, phi_a);
title('phi_a');
figure(6);
surf(X, Y, phi_b);
title('phi_b');
figure(7);
surf(X, Y, z_a);
title('z_a');
figure(8);
surf(X, Y, z_b);
title('z_b');
figure(9);
surf(X, Y, temper);
title('temper');
